pkg/eraser/eraser.go

```diff
@@ -205,14 +219,14 @@ func removeVulnerableImages() (err error) {
 	}
 
 	// TESTING :
-	r, err = listImages(backgroundContext, imageClient, "")
+	images, err = c.listImages(backgroundContext, "")
 	if err != nil {
 		return err
 	}
 
 	var allImages2 []string
```

Comment:
prealloc this one

Reply:
I get a message saying that value of images is never used when allocating images since it is set to c.listImages afterwards, so I allocated for allImages2